src/main/java/model/TaschenrechnerModel.java

```diff
@@ -69,22 +83,43 @@ private double divide(Operation operation) {
         return ans / enteredNumber;
     }
 
-    private double sin() throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+    private double sin() throws InvocationTargetException, IllegalAccessException, NoSuchMethodException {
```

Comment:
das kann man zusammenfassen und generalisieren:
`for (Class class : classesInPackage) {
    CalculatorFunction function = class;
    function.getCaption(); (oder function.calculate(param))
}`